src/Hooks/getAllExercises.ts

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+import { Exercise } from '../Datatypes/datatypes';
+
+const URL_API = 'localhost:8000';
```

Comment:
This should not be hardcoded but instead depend on a environment variable

Reply:
I could not make this work, I will try again - with the method you use on the Exercises microservice